proto/wippersnapper/signal/v1/signal.proto

```diff
@@ -17,12 +17,13 @@ import "wippersnapper/i2c/v1/i2c.proto";
 message I2CRequest {
   option (nanopb_msgopt).submsg_callback = true;
   oneof payload {
-    wippersnapper.i2c.v1.I2CBusInitRequest req_i2c_init                  = 1;
-    wippersnapper.i2c.v1.I2CBusScanRequest req_i2c_scan                  = 2;
-    wippersnapper.i2c.v1.I2CBusSetFrequency req_i2c_set_freq             = 3;
-    wippersnapper.i2c.v1.I2CDeviceInitRequest req_i2c_device_init        = 4;
-    wippersnapper.i2c.v1.I2CDeviceDeinitRequest req_i2c_device_deinit    = 5;
-    wippersnapper.i2c.v1.I2CDeviceUpdateRequest req_i2c_device_update    = 6;
+    wippersnapper.i2c.v1.I2CBusInitRequest req_i2c_init                     = 1;
+    wippersnapper.i2c.v1.I2CBusScanRequest req_i2c_scan                     = 2;
+    wippersnapper.i2c.v1.I2CBusSetFrequency req_i2c_set_freq                = 3;
+    wippersnapper.i2c.v1.I2CDeviceInitRequest req_i2c_device_init           = 4;
```

Comment:
Deprecate this one and always use lists? Or keep it and use it both ways?

Reply:
Keep it and use it both ways.

Reply:
@lorennorman  to elaborate - we'd want to use lists during synchronization only. Clicking "new component" will send the singular `I2CDeviceInitRequest`.